spec/rubocop/cop/rspec/repeated_example_group_body_spec.rb

```diff
@@ -226,6 +226,50 @@
     RUBY
   end
 
+  it 'skips `skip` and `pending` statements with arguments' do
+    expect_no_offenses(<<-RUBY)
+      context 'processed' do
+        skip 'some reason'
+      end
+
+      context 'processed' do
+        skip 'some reason'
+      end
+
+      context 'processed' do
+        pending 'another reason'
+      end
+
+      context 'processed' do
+        pending 'another reason'
+      end
+    RUBY
+  end
+
+  it 'registers offense correctly if `skip` and `pending` have block' do
+    expect_offense(<<-RUBY)
+      describe 'repeated' do
+      ^^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [5]
+        skip { blah }
```

Comment:
Is this a valid syntax? In a sense that the block seems to be ignored.

Might make sense to remove this example and only keep the one with the string argument?

Reply:
``` ruby
skip 'is skipped' do
  some_expect
end
# is same as
it 'is skipped' do
  skip
  some_expect
end
```

``` ruby
pending 'is pending' do
  some_expect
end
# is same as
it 'is pending' do
  pending
  some_expect
end
```

I added this test to represent explicitly that `skip` and `pending` with block cannot to be ignored.

Reply:
Aha, you're right, `skip`/`pending` are [also example methods](https://github.com/rspec/rspec-core/blob/5d87c28bb828ba14b54a0d8cfddf044894009314/lib/rspec/core/example_group.rb#L195), and we [do account for that](https://github.com/rubocop-hq/rubocop-rspec/blob/300943ef5ca71d8708685c17c10f3d95b6509b04/lib/rubocop/rspec/language.rb#L87).